bucket/microsoft16.json

```diff
@@ -0,0 +1,20 @@
+{
+    "description": "The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft's new way to collaborate and contribute to the Java ecosystem.",
+    "homepage": "https://www.microsoft.com/openjdk",
+    "version": "16.0.1-9",
+    "license": {
+        "identifier": "GPL2 with Classpath Exception",
```

Comment:
Perhaps this should be `GPL-2.0-with-classpath-exception` per 
https://github.com/lukesampson/scoop/wiki/App-Manifests#optional-properties and 
https://spdx.org/licenses/ ?